frontend/src/components/place/PlaceSidebarInfo.vue

```diff
@@ -85,7 +85,7 @@
             </div>
             <template v-if="place.features.length > 0">
                 <div class="place-sidebar__features">
-                    <h2 class="feature-title">Features</h2>
+                    <h2 class="block-title">Features</h2>
```

Comment:
Move "Features" to the localization constant, and other phrases in this component also